descriptors/ModuleDescriptor-template.json

```diff
@@ -795,6 +795,30 @@
           "delay": "1"
         }
       ]
+    },
+    {
+      "id": "_tenant",
+      "version": "1.0",
+      "interfaceType": "system",
+      "handlers": [
+        {
+          "methods": [
+            "POST"
+          ],
+          "pathPattern": "/_/tenant",
+          "modulePermissions": [
+            "pubsub.event-types.post",
+            "pubsub.publishers.post",
+            "pubsub.subscribers.post"
+          ]
+        },
+        {
+          "methods": [
+            "DELETE"
+          ],
+          "pathPattern": "/_/tenant"
+        }
```

Comment:
Seems removal is not implemented, can it be removed?

Reply:
Removed

Reply:
Should disabling a tenant be supported?

Are publishers registered with pub-sub? If so, should they be unregistered?

Reply:
That's a good question. PubSub readme doesn't mention "unregistration", but there's a DELETE endpoint. It looks from our experience with vagrant box that nothing bad should happen if we don't unregister, but I don't know if it was intended by PubSub module creators.

Reply:
According to @TarasSpashchenko we're supposed to implement DELETE /_/tenant and clean up our registrations. This PR will be updated.

Reply:
Updated - DELETE is handled now